src/components/Sheet/README.md

```diff
@@ -13,6 +13,7 @@ keywords:
   - dialog
   - web
 fullSizeExamples: true
+deprecationNotice: The Sheet component encourages designers to create a new layer on top of the page instead of improving the existing user interface. It also blocks other parts of the UI, forces users to switch context and adds complexity to otherwise simple interactions.
```

Comment:
```suggestion
deprecationNotice: The sheet component encourages designers to create a new layer on top of the page instead of improving the existing user interface. It also blocks other parts of the UI, forces users to switch context, and adds complexity to otherwise simple interactions.
```